CHANGELOG.rst

```diff
@@ -57,6 +57,9 @@ Added
         from fonduer.parser import Parser
         parser = Parser(session, vizlink=CustomVisualLinker())
 
+* `@HiromuHota`_: Add `LingualParser`, which any lingual parser like `Spacy` should inherit from,
+  and add `lingual_parser` as an argument to `Parser` to be able to plug a custom lingual parser.
+
```

Comment:
Can we update CHANGELOG as well?

Reply:
I added some changes to CHANGELOG but I didn't include changes that do not affect how to use Fonduer like additional tests, doc section, etc.